services/web-app/pages/developing/index.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Developing
+description:
+---
+
+<PageDescription>
+  Welcome to Carbon! If you’re just started out developing with Carbon, here’s a checklist of
+  everything you need to get up and running.
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Step 1: Learn about Carbon</AnchorLink>
+  <AnchorLink>Step 2: Get started</AnchorLink>
+  <AnchorLink>Step 3: Explore developer tools</AnchorLink>
+</AnchorLinks>
+
+## Step 1: Learn about Carbon
+
+Carbon translates the [IBM Design Language](http://www.ibm.com/design/language) into resources for
+building digital experiences. Take the time to read the site so that you fully understand what
+drives IBM’s design philosophy and principles, and can make informed decisions in your product work.
+
+By adopting Carbon, front-end developers and engineers gain a collection of reusable components to
+build websites and user interfaces quickly and efficiently.
+
+### Understand the ecosystem
+
+The following diagram shows the relationships of assets and guidance across IBM. The Carbon Design
+System is often extended or complimented by additional product and digital systems.
+
```

Comment:
are we missing a picture here? 
<img width="899" alt="image" src="https://user-images.githubusercontent.com/40550942/171468399-252cb8c8-78f5-4482-8e65-ad4d8bb70e10.png">